co2eq/purchase/footprints.yml

```diff
@@ -2201,7 +2201,7 @@ _children:
             unit: kg
             conversions:
               L:
-                kilograms: 75
+                kilograms: 1.34
```

Comment:
Can you maybe add a link/reference to the issue we're fixing here?

Reply:
sure, i was just drafting the "main" internal PR with all the relevant information + migration. But direct issue is here: https://github.com/tmrowco/tmrow/issues/2023